synapse/tools/superhash.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+synapse - superhash.py
+Created on 4/26/17.
+
+Generate the hashsets and guid (superhash) for a file.
+"""
+# Stdlib
+import argparse
```

Comment:
style guide (visi ocd).  please order imports by ascending length :D.  Adherence to ridiculous style nits keeps the codebase in "one voice".